packages/lab/src/badge/Badge.css

```
@@ -24,44 +6,35 @@
   position: relative;
   flex-shrink: 0;
   vertical-align: middle;
```

I would replace this with 
```suggestion
  justify-content: center;
  align-items: center;
```
So we can center the content of the badge